dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -820,4 +822,12 @@ public void delete(String name) {
 
         return mergedSettings;
     }
+
+    public Map<String, String> getFileSettingsForPrefix(String prefix) {
```

Comment:
I'm not sure if this is the right place for obtaining onelogin properties. It doesn't have "full chain" of obtaining configurable values (for example db support)

Reply:
Yes, however it's explicitly stated in the name of the method, that these are file-based settings, that will be returned – do you think that it's not sufficiently clear or maybe I'm missing something?